shared/checks/oval/only_listed_users_allowed_in_etc_passwd.xml

```diff
@@ -0,0 +1,43 @@
+<def-group>
+  <definition class="compliance" id="only_listed_users_allowed_in_etc_passwd" version="1">
+    <metadata>
+      <title>Only Listed Local Users Exist on Operating System</title>
+      <affected family="unix">
+        <platform>Oracle Linux 7</platform>
+      </affected>
+      <description>Besides the default operating system users, there should be no other users
+      except the users in the given list that exist locally on the operating system.</description>
+    </metadata>
+    <criteria>
+      <criterion test_ref="test_only_listed_users_allowed_in_etc_passwd"
+      comment="only root user and explicitly listed users are allowed in /etc/passwd" />
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test id="test_only_listed_users_allowed_in_etc_passwd" version="1"
+  check_existence="any_exist" check="all" comment="query /etc/passwd">
+    <ind:object object_ref="object_only_listed_users_allowed_in_etc_passwd" />
+    <ind:state state_ref="state_only_listed_users_allowed_in_etc_passwd" />
+  </ind:textfilecontent54_test>
+
+  <ind:textfilecontent54_object id="object_only_listed_users_allowed_in_etc_passwd" version="1">
+    <ind:filepath>/etc/passwd</ind:filepath>
+    <ind:pattern>^([a-zA-Z0-9_.-]+?):</ind:pattern>
```

Comment:
Sorry I didn't get this before, but `pattern` element requires `operation="pattern match"`